### ISSUE TITLE
Cron module with state=absent deletes entire crontab file

### ISSUE THREAD

**Issue by jkman340** [ansible/ansible#73531]
##### SUMMARY
When using the cron module with state=absent, the entire /etc/crontab file is deleted vs just ensuring the specified cron entry is not present. 

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
ansible.builtin.cron

##### ANSIBLE VERSION
```
ansible 2.10.5
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.9 (default, Oct  8 2020, 12:12:24) [GCC 8.4.0]
```

##### CONFIGURATION

```
ANSIBLE_PIPELINING(/etc/ansbile/ansible.cfg) = True

```

##### OS / ENVIRONMENT
Ubuntu 18.04

##### STEPS TO REPRODUCE
Run a playbook with a cron job with state absent

```
- name: Ensure job does not exist on this server
  ansible.builtin.cron:
    name: "Sample job name"
    user: root
    cron_file: /etc/crontab
    state: absent
    job: "/usr/local/bin/sample.py"
    minute: "0"
    hour: "23"
    day: "*"
    month: "*"
    weekday: "*"
```

##### EXPECTED RESULTS

Cron job is removed/not present in cron file

##### ACTUAL RESULTS

Entire /etc/crontab file is deleted

```paste below

```

**Comment by sivel** [comment]
Duplicate of https://github.com/ansible/ansible/issues/72392

**Comment by jkman340** [comment]
Reading thru #72392 this behavior still seems unexpected. #72392  identifies an issue with having multiple jobs in a single /etc/cron.d file.  This issue relates to managing jobs in the main /etc/crontab and the entire file being deleted when trying to remove a single job. Is this not supported?

**Comment by samdoran** [comment]
This comes up enough that we should special case `/etc/crontab`. This should be an error rather than removing the file.

**Comment by samdoran** [comment]
Fixed by #74497.
